R/indentify.r

```diff
@@ -120,14 +111,12 @@ setMethod("detect_flightline", "LAS",
 #' @return Return nothing. The original object is modified in place by reference.
 #' @export detect_scanline
 #' @importFrom data.table setorder
-setGeneric("detect_scanline", function(obj){standardGeneric("detect_scanline")})
+setGeneric("detect_scanline", function(obj, dt = 30){standardGeneric("detect_scanline")})
 
 #' @rdname detect_scanline
 setMethod("detect_scanline", "LAS",
   function(obj)
   {
-    gpstime <- scanlineID <- ScanDirectionFlag <- NULL
```

Comment:
I don't remember suppressing the NULL initialisation neither modifying the function detect_scanline (my memory or an old version of the code??), but I should have noticed that when committing, I'll be more careful next time... Thank you for the tip on the NULL initialization, I was wondering why.
For the gpstime uniqueness I had the same doubt, but I checked on all the datasets I had and it seems to have a unique gpstime value for al echos of the same pulse, confirmed by that forum [https://groups.google.com/forum/#!msg/lastools/VWV4DVkiVq8/EIs9Jvjd8KIJ](https://groups.google.com/forum/#!msg/lastools/VWV4DVkiVq8/EIs9Jvjd8KIJ).